[RHACS] Improve docs for RHACS OpenShift OAuth provider (#45324)
openshift/openshift-docs: operating/manage-user-access/configure-ocp-oauth.adoc

----
$ oc -n stackrox set env deploy/central ROX_ENABLE_OPENSHIFT_AUTH=true
----

> * For access rules, the key "Name" should be used to reference the user name returned from the {ocp} OAuth server.

Author: Compared to other authentication providers we provide, the openshift auth one doesn't provide a test feature with which you can 
- test login
- review the returned values for i.e. `userid`, `name`, `groups`

Do you think it's worthwhile to have an example here that would showcase what value within the configuration would be mapped to `userid`, `name`, `groups` respectively @gaurav-nelson ?

Reviewer: Thank you @dhaus67 yes, I agree having an example would be helpful indeed.

Author: I've added some sample values now, let me know what you think.

----
$ oc -n stackrox set env deploy/central ROX_ENABLE_OPENSHIFT_AUTH=true
----

> * For access rules, the key "Name" should be used to reference the user name returned from the {ocp} OAuth server.

Reviewer: ```suggestion
* For access rules, you should use the key `Name` to reference the user name that the {ocp} OAuth server returns.
```

> * For access rules, the key "Email" will not be returned from the {ocp} OAuth server.

Reviewer: ```suggestion
* For access rules, the {ocp} OAuth server does not return the key `Email`.
```
